src/style.css

```diff
@@ -1,3 +1,71 @@
+:root {
+  --letters-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  background-color: gray;
+  text-transform: uppercase;
+}
+
+.header {
+  width: auto;
+  background-color: #fff;
+  padding-left: 50px;
+  padding-right: 50px;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  align-items: center;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: block;
```

Comment:
Consider using 'display: flex;' for the '.nav' class to maintain consistency with the '.nav_list' class and potentially improve layout flexibility.